build.gradle

```diff
@@ -24,7 +24,7 @@ plugins {
 ext {
     genDir             = file('src/gen')
     sharedDir          = file("${project.rootDir}/shared")
-    sailfishVersion     = '3.2'
+    sailfishVersion     = '3.2-SNAPSHOT'
```

Comment:
Using snapshot libraries is not a good practice.
It will result different builds and behavior depending when the build will be done

Reply:
Yes, we all aware of it. But it us the only way for now